game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/apeatroll/daga.plugin.kts

```diff
@@ -0,0 +1,87 @@
+package gg.rsmod.plugins.content.areas.apeatroll
+
+import gg.rsmod.plugins.content.mechanics.shops.CoinCurrency
+
+/**
+ * @author Alycia <https://github.com/alycii>
+ */
+
+create_shop("Daga's Scimitar Smithy", CoinCurrency(), containsSamples = false, purchasePolicy = PurchasePolicy.BUY_STOCK) {
+    items[0] = ShopItem(Items.BRONZE_SCIMITAR, 10)
+    items[1] = ShopItem(Items.IRON_SCIMITAR, 10)
+    items[2] = ShopItem(Items.STEEL_SCIMITAR, 8)
+    items[3] = ShopItem(Items.MITHRIL_SCIMITAR, 6)
+    items[4] = ShopItem(Items.DRAGON_SCIMITAR, 0)
+
+}
+
+//If the player has not yet completed Monkey Madness I,
+// there will be 0 stock of dragon scimitars.
+
+
+on_npc_option(npc = Npcs.DAGA, option = "talk-to") {
+    //missing function greegree
```

Comment:
Please use Todo or Fixme comments so that we can flag files we need to come back to later.

https://www.jetbrains.com/help/idea/using-todo.html